src/Service.Host/client/src/components/reportOptions/SalesArticleCoreTypesOptions.js

```diff
@@ -0,0 +1,46 @@
+﻿import React, { Component } from 'react';
+import { Button, Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import Page from '../../containers/Page';
+
+class SalesArticleCoreTypesOptions extends Component {
+    constructor(props) {
```

Comment:
Not using state so could be a function component?

Reply:
good idea